sdks/python/apache_beam/transforms/util.py

```diff
@@ -86,16 +86,16 @@ class CoGroupByKey(PTransform):
                     'tag2': ... ,
                     ... })
 
-  For example, given:
+  For example, given::
 
       {'tag1': pc1, 'tag2': pc2, 333: pc3}
 
-  where:
+  where::
```

Comment:
I believe we also need a blank line after this based on https://www.sphinx-doc.org/en/1.5/rest.html#source-code

Reply:
Done, thank you!